ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableDataProvider.kt

```diff
@@ -0,0 +1,57 @@
+package com.revenuecat.purchases.ui.revenuecatui.data.processed
+
+import android.content.Context
+import com.revenuecat.purchases.Package
+import java.text.NumberFormat
+import java.util.Currency
+import java.util.Locale
+
+@Suppress("UnusedParameter", "FunctionOnlyReturningConstant")
+internal object VariableDataProvider {
```

Comment:
This could really use tests.

Reply:
Added in https://github.com/RevenueCat/purchases-android/pull/1251/commits/3bc75362d76428f1106968040c536e0fbaad3d9a